node-proxy/lib/utils/http-utils.js

```diff
@@ -29,12 +29,12 @@ exports.createProtocolServer = function(protocol, opts) {
       break;
     case 'https':
       var ssl_opts  = {
-        secureProtocol: 'SSLv23_method',
         secureOptions: constants.SSL_OP_NO_SSLv3
       };
       ssl_opts.ca   = fs.readFileSync(opts.ca);
       ssl_opts.cert = fs.readFileSync(opts.certificate);
       ssl_opts.key  = fs.readFileSync(opts.private_key);
+      ssl_opts.ciphers = opts.ciphers;
```

Comment:
It looks like a default is set, so for that to happen an admin would have
to explicitly set that.
On Jan 20, 2016 5:24 PM, "Miciah Dashiel Butler Masters" <
notifications@github.com> wrote:

> In node-proxy/lib/utils/http-utils.js
> <https://github.com/openshift/origin-server/pull/6349#discussion_r50330124>
> :
>
> >          secureOptions: constants.SSL_OP_NO_SSLv3
> >        };
> >        ssl_opts.ca   = fs.readFileSync(opts.ca);
> >        ssl_opts.cert = fs.readFileSync(opts.certificate);
> >        ssl_opts.key  = fs.readFileSync(opts.private_key);
> > +      ssl_opts.ciphers = opts.ciphers;
>
> What happens if opts.ciphers is not defined in web-proxy-config.json?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-server/pull/6349/files#r50330124>.
>